tests/key_backup_test.go

```diff
@@ -50,7 +50,10 @@ func TestCanBackupKeys(t *testing.T) {
 			// Now login on a new device
 			csapiAlice2 := tc.MustRegisterNewDevice(t, tc.Alice, "BACKUP_RESTORER")
 			backupRestorer := tc.MustLoginClient(t, &cc.ClientCreationRequest{
-				User: csapiAlice2,
+				User: &cc.User{
+					CSAPI:      csapiAlice2.CSAPI,
+					ClientType: clientTypeB,
```

Comment:
This was lost in the refactor to how clients are made. I need to check where else this was forgotten.